### ISSUE TITLE
Issue with alias resolver

### ISSUE THREAD

**Issue by a-tonchev** [gxmari007/vite-plugin-eslint#9]
I am trying to use aliases, usually it works with the https://www.npmjs.com/package/eslint-import-resolver-alias

But when I run the app, the plugin is trowing all the time:

`Unable to resolve path to module '@/components/....'`

It seems that adding 

```
  "settings": {
    "import/resolver": {
      "alias": {
        "map": [
          ["@", "./src"]
        ]
      }
    }
  },
```

Is not detected by the vite plugin

**Comment by a-tonchev** [comment]
Sorry, my mistake

**Comment by pzmosquito** [comment]
I know this is an old issue, but here is an alternative: https://www.npmjs.com/package/eslint-import-resolver-vite
